app/core/modals/settings.tsx

```diff
@@ -1,6 +1,8 @@
 import { Dialog, Transition, Tab } from "@headlessui/react"
 import { Fragment, useState } from "react"
 import { Close } from "@carbon/icons-react"
+import { useRecoilState, useResetRecoilState } from "recoil"
```

Comment:
Just removing unused dependency

```suggestion
import { useRecoilState } from "recoil"
```